language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/CompletionTest.java

```diff
@@ -86,21 +84,22 @@ public void test(String config, String configPath) throws WorkspaceDocumentExcep
         }
     }
 
-    String getResponse(JsonObject configJsonObject) throws IOException {
+    private String initializeSources(JsonObject configJsonObject) throws IOException {
```

Comment:
Better if we can change the method name since the name does not reflect what it actually does.

Reply:
Shall we change the method name to another `getResponse`?